galaxy_ng/app/migrations/_dab_rbac.py

```diff
@@ -39,18 +42,32 @@ def split_pulp_roles(apps, schema_editor):
     GroupRole = apps.get_model('core', 'GroupRole')
 
     for corerole in Role.objects.all():
+        print(f'ROLE {corerole} {corerole.name}')
         split_roles = {}
         for assignment_cls in (UserRole, GroupRole):
+            print(f'\t{assignment_cls}')
             for pulp_assignment in assignment_cls.objects.filter(role=corerole, content_type__isnull=False):
+                print(f'\t\t{assignment_cls} {pulp_assignment}')
```

Comment:
If you're going to leave these (which would be okay) then padding it with more detail would be helpful.

Reply:
not planning to leave any new prints ... just waiting for all tests to pass and all issues to get discovered before the final cleanup